src/schemas/21P-530/schema.js

```diff
@@ -34,6 +34,42 @@ let schema = {
   type: 'object',
   additionalProperties: false,
   definitions: {
+    address: {
```

Comment:
This seems like it's going to have implications for prefilling this form, @markolson?